beacon-chain/blockchain/forkchoice/metrics.go

```diff
@@ -0,0 +1,4 @@
+/*
+Package forkchoice TODO(3219): Add forking metrics.
```

Comment:
No need for multiple `Package forkchoice` statements. It will appear multiple times at the top of the godoc